### PR TITLE
Added sphinx question

### PR DESCRIPTION
**Pull request by bmwoodruff**
Have fun

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,3 +4,6 @@ A place to practice testing on github.
 Making changes to push to my forked versions.
 
 Now I want to practice adding things. 
+
+Looks like I'm getting the hang of things.  I need to practice links now, using sphinx markup.
```
Comment (mathmanican) [inline README.md:8]:
```suggestion
Looks like I'm getting the hang of things.  I need to practice links now, using sphinx markup. 
```
That should finish things up.

Comment (mathmanican) [inline README.md]:
```suggestion
Looks like I'm getting the hang of things. 
```

Comment (mathmanican) [reply README.md]: Adding comment after review

File: README.md [hunk 2]
```diff
@@ -4,3 +4,6 @@ A place to practice testing on github.
 Making changes to push to my forked versions.
 
 Now I want to practice adding things. 
+
+Looks like I'm getting the hang of things. 
+
```
Comment (bmwoodruff) [inline README.md]:
```suggestion
And now we add a link like so....
```

**Review by bmwoodruff — COMMENTED** [review]
added link

**Pull request merged**